classes/class-p4-control-panel.php

```diff
@@ -145,32 +145,31 @@ public function check_cache() {
 			// If this is an ajax call.
 			if ( wp_doing_ajax() ) {
 				// Allow this action only to Administrators.
-				if ( ! current_user_can( 'manage_options' ) ) {
-					return;
-				}
-				$cp_nonce  = filter_input( INPUT_GET, '_wpnonce',  FILTER_SANITIZE_STRING );
-				$cp_action = filter_input( INPUT_GET, 'cp-action', FILTER_SANITIZE_STRING );
-
-				// CSRF check and action check.
-				if ( wp_verify_nonce( $cp_nonce, 'cp-action' ) && 'check_cache' === $cp_action ) {
-					$response = [];
-					$info     = wp_redis_get_info();
-
-					if ( $info instanceof WP_Error ) {
-						if ( $info->errors['wp-redis'] && is_array( $info->errors['wp-redis'] ) ) {
-							$response['message'] = $info->errors['wp-redis'][0];
-							$response['class']   = 'cp-error';
+				if ( current_user_can( 'manage_options' ) || current_user_can( 'editor' ) ) {
```

Comment:
Just a suggestion here. You can keep the old logic, just to prevent indenting one level more.
`if ( ! current_user_can( 'manage_options' ) && ! current_user_can( 'editor' )  ) { return; }`

Reply:
Done!